homeassistant/components/sensor/dweet.py

```diff
@@ -17,7 +22,14 @@
 REQUIREMENTS = ['dweepy==0.2.0']
 
 DEFAULT_NAME = 'Dweet.io Sensor'
-CONF_DEVICE = 'device'
+
+PLATFORM_SCHEMA = vol.Schema({
+    vol.Required(CONF_PLATFORM): 'dweet',
+    vol.Required(CONF_DEVICE): cv.string,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_UNIT_OF_MEASUREMENT): cv.string,
+    vol.Optional(CONF_VALUE_TEMPLATE): cv.template,
```

Comment:
Based on the diff, it seems like value template was required but is now optional. Is this intentional?